spec/classes/pulpcore_spec.rb

```diff
@@ -51,6 +51,29 @@
           is_expected.to contain_exec('pulpcore-manager reset-admin-password --random')
         end
 
+        it 'does not install Redis' do
+          is_expected.not_to contain_class('redis')
+        end
+
+        it 'does not configure Pulpcore connection to Redis' do
+          is_expected.to contain_concat__fragment('base')
+            .without_content(/REDIS_HOST/)
```

Comment:
It's a bit of a bikeshed, but for consistency within the file the `%r{}` is used for regex already and I want to say that is preferred by the linter (I swear something yelled at me recently for not using it).

Reply:
Indeed you're correct.

I thought we were mixing `//` and `%r{}` already in this file, BUT it turns out the instances of `//` I saw are only the ones I introduced.

I'll update to use `%r{}` consistently for now, and we can work out in the larger test PR I have planned whether one or both styles should actually be used.